WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -416,6 +417,10 @@ extension AddAttributeOptionsViewController {
         present(actionSheet, animated: true)
     }
 
+    func trackRenameAttributeButtonTapped() {
+        analytics.track(event: WooAnalyticsEvent.Variations.renameAttributeButtonTapped(productID: self.viewModel.product.productID))
```

Comment:
nit: `self` is not needed since we are not in a closure anymore!

Reply:
Someday I will catch all of the `self`s when I cut/paste things to move them out of closures 😆